.vscode/launch.json

```
@@ -184,6 +184,26 @@
 				"order": 4
 			}
 		},
+		{
+			// Note this uses the current Code/Positron executable.
+			// Launch from a dev version if you need dev Positron.
```

```suggestion
			// Note this uses the version of Code/Positron you launch it from.
			// i.e., launch from dev Positron if your tests need dev Positron.
```

A little more explicitly here